pkg/volume/image/image.go

```diff
@@ -67,8 +67,8 @@ func (o *imagePlugin) ConstructVolumeSpec(volumeName, mountPath string) (volume.
 func (o *imagePlugin) GetAttributes() volume.Attributes {
 	return volume.Attributes{
 		ReadOnly:       true,
-		Managed:        true,
```

Comment:
Please double-check for SELinux management, do we need to reset them to `true`? @saschagrunert

Reply:
I think `Managed` only enables this code path: 

https://github.com/kubernetes/kubernetes/blob/92507eaabe5c93158f64faca240d02d0e5875a01/pkg/kubelet/kubelet_pods.go#L290-L293

This makes it possible to set the `SelinuxRelabel` field for the runtime in:

https://github.com/kubernetes/kubernetes/blob/92507eaabe5c93158f64faca240d02d0e5875a01/pkg/kubelet/kuberuntime/kuberuntime_container.go#L442-L451

In CRI-O we don't need to relabel because we can mount with the right label from the process. Means that setting it to `false` seems reasonable.

I assume the e2e tests fail if we set this field to `true`?